src/App.vue

```diff
@@ -31,8 +32,10 @@ import MailboxLockedError from './errors/MailboxLockedError'
 
 export default {
 	name: 'App',
-	mounted() {
+	async mounted() {
 		this.sync()
+		await this.$store.dispatch('fetchCurrentUserPrincipal')
+		await this.$store.dispatch('loadCollections')
```

Comment:
out of interest - will this initialise a DAV client for every message? If so, is this problematic or could we initialise it only on detecting a iMIP message?

Reply:
The client is only initialized once and then cached.